### PR TITLE
Search directory path for a `.pylintrc` config file

### PR DESCRIPTION
**Pull request by nigef**
Check for `.pylintrc` file provided in user location (i.e. the student starter code). Does not search upward, in parent directories.

In the case where `_check` is called on a directory and files are checked recursively, a cache of pylintrc locations is _not_ used for now, so there is more work done loading a config files.

Aside: although different from our approach, for background info on how pylint finds a `.pylintrc` file, see, `pylint.config.find_pylintrc`

### CODE REVIEW

**Comment by nigef** [comment]
RE: 1. I don't see a realistic way around this.

**Comment by nigef** [comment]
RE: 4. cache removed! aside: I'm curious why we don't want it.

File: python_ta/__init__.py [hunk 1]
```diff
@@ -155,7 +129,7 @@ def _verify_pre_check(filepath):
         # Check for inline "pylint:" comment, which may indicate a student
         # trying to disable a check.
         with tokenize.open(filepath) as f:
-            for (tok_type, content, _, _, _) in tokenize.generate_tokens(f.readline):
+            for (tok_type,content,_,_,_) in tokenize.generate_tokens(f.readline):
```
Comment (david-yz-liu) [inline python_ta/__init__.py]: Please revert this change.

**Comment by david-yz-liu** [comment]
For (1), you could create a new linter for each file.
For (4), in general caching introduces an extra level of complexity that should be avoided until deemed necessary.

**Comment by nigef** [comment]
ready to merge?

**Comment by david-yz-liu** [comment]
I still see a problem with the "#1" I pointed out earlier.
